registration-processor-default.properties

```diff
@@ -478,13 +478,22 @@ mosip.biometric.sdk.providers.face.mosip-ref-impl-sdk-client.format.url.mock-1.1
 mosip.registration.processor.credentialtype=euin
 mosip.registration.processor.encrypt=false
 mosip.registration.processor.credential.request.service.id=mosip.credential.request.generator 
-mosip.registration.processor.digitalcard.credentialtype=PDFCard
-mosip.registration.processor.print.credentialtype=euin
-mosip.registration.processor.digitalcard.template=RPR_UIN_CARD_TEMPLATE
-mosip.registration.processor.print.template=RPR_UIN_CARD_TEMPLATE
-
-# this property is used to define issuers eg partnerid:credentialType:templateTypeCode
-mosip.registration.processor.issuer=mpartner-default-print:${mosip.registration.processor.print.credentialtype}:${mosip.registration.processor.print.template};mpartner-default-digitalcard:${mosip.registration.processor.digitalcard.credentialtype}:${mosip.registration.processor.digitalcard.template}
+# This property is used to define default internal credential issuers separated by comma.
+# eg: partnerid#credentialType#templateTypeCode
+mosip.registration.processor.default.internal.issuers=mpartner-default-digitalcard#PDFCard#RPR_UIN_CARD_TEMPLATE
+# This property is used to define default issuers separated by comma,
+# eg: partnerid#credentialType#templateTypeCode
+mosip.registration.processor.default.issuers=mpartner-default-print#euin#RPR_UIN_CARD_TEMPLATE
+
+# Credential Partner Configuration
+# First language defined in the mandatory language list will be considered to identify the attribute value.
+mosip.mandatory-languages=eng
```

Comment:
There is already property in application-default properties file, we have to use that same in the code and not define separate
https://github.com/mosip/mosip-config/blob/develop-v3/application-default.properties#L408

Reply:
yes, not required here.. removed.